src/views/routes/details/components/tabs/detailsTab/components/RouteDetailsSection/DetailsSectionRightColumn.tsx

```diff
@@ -1,5 +1,11 @@
 import React, { FC } from 'react';
 
+import {
+  DescriptionList as DL,
```

Comment:
Is there a reason why you aliased these to abbreviations here, but didn't do it in other places? There's nothing wrong with it, of course, as long as we're consistent.